### ISSUE TITLE
add emoji support

### ISSUE THREAD

**Issue by krombel** [QMatrixClient/Quaternion#255]
showing them but writing them as well

**Comment by rushsteve1** [comment]
If you have an Emoji font installed it does display them.

**Comment by KitsuneRal** [comment]
Having an emoji font installed is supposed to be the proper solution for displaying, of course. Writing is the much harder part.

**Comment by tmathews** [comment]
What about an option for supporting writing with colon codes as a jump start? :thumbs_up:

**Comment by rushsteve1** [comment]
I second what @tmathews says. Colon codes are easy and used by many other chat apps.
As for an Emoji font... why not just depend/suggest one when we build packages?

**Comment by tmathews** [comment]
I agree with using the system emoji font - enforces OS creators to include it. Though being able to override is nice, the extra work is not deemed necessary.
